src/website/src/pages/ec/components/banner/docs/code.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Showcase
+order: 2
+---
+
+import { Html, Playground } from '@ecl/website-components';
+import {
+  bannerImageBox,
+  bannerImageShade,
+  bannerImageGradient,
+  bannerPrimary,
+} from '../demo';
+
+## Image banners
+
```

Comment:
Maybe this was requested exactly like this, but since the default variant is the "primary" i wonder the reason why it is the last in the showcase...